data/companies.json

```
@@ -176,6 +176,22 @@
         "youtube": "https://www.youtube.com/c/CloudfactoryHQ"
       }
     },
+    {
+        "name": "Clusus Information Solutions",
+        "establishedAt": 1535414400,
+        "location": "Jhamsikhel, Lalitpur",
+        "website": "https://www.clusus.com/",
+        "email": "Info@clusus.com",
+        "phone": "+977-01-5400593",
```

The phone number formatting is incorrect.
```suggestion
        "phone": "+977-15400593",
```